From 9dfaba553cb5aa0c0d689de1555a3bb45ae7dd3a Mon Sep 17 00:00:00 2001
From: Alexey Shvetsov <alexxyum@gmail.com>
Date: Thu, 21 Nov 2024 10:56:17 +0300
Subject: [PATCH] Fix build with gcc-15
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Error is
energyhistory.h:89:5: error: ‘int64_t’ does not name a type
   89 |     int64_t             nsteps;       //! The number of steps in the history
      |     ^~~~~~~

Ref: https://bugs.gentoo.org/939006

Signed-off-by: Alexey Shvetsov <alexxyum@gmail.com>
---
 src/gromacs/mdtypes/energyhistory.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gromacs/mdtypes/energyhistory.h b/src/gromacs/mdtypes/energyhistory.h
index 0c8fecfae7..f80d4c56b9 100644
--- a/src/gromacs/mdtypes/energyhistory.h
+++ b/src/gromacs/mdtypes/energyhistory.h
@@ -47,6 +47,8 @@
 #ifndef GMX_MDLIB_ENERGYHISTORY_H
 #define GMX_MDLIB_ENERGYHISTORY_H
 
+#include <cstdint>
+
 #include <memory>
 #include <vector>
 
-- 
2.45.2

